### PR TITLE
Improved Windows setup

### PR DESCRIPTION
**Pull request by Jan-LucaKlees**
To make this a fully featured setup, I added some code to the installer, that will set the %PATH% variable in Windows and asks for a restart after the setup. This way we make sure that the CLI will be accessible from the CMD. Further information can be found in the commit notes.

### CODE REVIEW

**Comment by Jan-LucaKlees** [comment]
Just tested it on a VM running 64 bit Windows, it's working.

File: build/innosetup/phraseapp-client.iss [hunk 1]
```diff
@@ -24,10 +24,36 @@ Compression=lzma
...
 
 [Files]
-Source: "../../dist/phraseapp_windows_amd64.exe"; DestDir: "{app}\phraseapp"; DestName: "phraseapp.exe"; Flags: ignoreversion
+Source: "../../dist/phraseapp_windows_amd64.exe"; DestDir: "{app}"; DestName: "phraseapp.exe"; Flags: ignoreversion
 ; NOTE: Don't use "Flags: ignoreversion" on any shared system files
 
+[Registry]
+Root: HKLM; Subkey: "SYSTEM\CurrentControlSet\Control\Session Manager\Environment"; \
+    ValueType: expandsz; ValueName: "Path"; ValueData: "{olddata};{app}"; \
+    Check: NeedsAddPath('C:\foo')
```
Comment (theSoenke) [inline build/innosetup/phraseapp-client.iss]: @Jan-LucaKlees Stimmt das `foo` hier so?

Comment (Jan-LucaKlees) [reply build/innosetup/phraseapp-client.iss]: Upps

**Comment by Jan-LucaKlees** [comment]
@theSoenke I think this is as good as it gets. Feel free to test a bit around and release it.

**Comment by theSoenke** [comment]
@Jan-LucaKlees Thanks for adding it :+1:

**Pull request merged**